video/out/drm_common.c

```diff
@@ -497,19 +499,83 @@ static bool setup_mode(struct kms *kms, const char *mode_spec)
     return false;
 }
 
-static int open_card(int card_no)
+static int open_card_path(char *path)
```

Comment:
```suggestion
static int open_card_path(const char *path)
```

Reply:
Actually just went through const'ification of these related things. So consider this done.